tutorials/16_implemented_methods.ipynb

```diff
@@ -255,6 +255,45 @@
     "    proposal = posterior"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e7ecf9a0-6a82-42e1-9c12-97d0f5d825be",
+   "metadata": {},
+   "source": [
+    "**Contrastive Neural Ratio Estimation**<br>Benjamin Kurt Miller, Christoph Weniger, Patrick Forré (NeurIPS 2022) <br>[[PDF]](https://arxiv.org/pdf/2210.06170.pdf)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ec55e76-dd86-46d1-a7cc-643324488820",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# One of the features of NRE-C is producing an ratio of densities at optimum even with multiple classes.\n",
+    "# This only occurs when the algorithm is amortized (non-sequential).\n",
+    "\n",
+    "from sbi.inference import SNRE_C\n",
+    "\n",
+    "# Amortized inference\n",
+    "inference = SNRE_C(prior)\n",
+    "proposal = prior\n",
+    "theta = proposal.sample((num_sims,))\n",
+    "x = simulator(theta)\n",
+    "_ = inference.append_simulations(theta, x).train()\n",
+    "posterior = inference.build_posterior().set_default_x(x_o)\n",
+    "\n",
+    "# Sequential inference (no exact density ratio guarantee at optimum).\n",
+    "inference = SNRE_C(prior)\n",
+    "proposal = prior\n",
+    "for _ in range(rounds):\n",
+    "    theta = proposal.sample((num_sims,))\n",
+    "    x = simulator(theta)\n",
+    "    _ = inference.append_simulations(theta, x).train()\n",
+    "    posterior = inference.build_posterior().set_default_x(x_o)\n",
+    "    proposal = posterior"
+   ]
+  },
```

Comment:
I would really like for this file to only contain the algorithms described in the respective papers. I would thus suggest to remove the sequential method (same problem for SNRE_A, where we also did not add the sequential method to the tutorial).

Reply:
sounds good!